ui/eureka/eureka.js

```diff
@@ -2,6 +2,10 @@
 
 let Options = {
   PopSound: '../../resources/sounds/PowerAuras/sonar.ogg',
+  PopVolume: 1.0,
+  TrackBunnyFates: true,
```

Comment:
I think it'd be fine to always track bunny fates and not give an option, for what it's worth.  If people don't like it, they can turn off the sound at least.